calico/etcddriver/protocol.py

```diff
@@ -150,6 +151,10 @@ def new_messages(self, timeout=1):
         try:
             data = self._sck.recv(16384)
         except socket.error as e:
+            #PLW: Should we return or retry in line? Concluded this is OK,
+            # but should comment that can return immediately with 0 messages
+            # in some error cases, which implies that you should retry.
+            # PLW: EAGAIN and EWOULDBLOCK are the same thing.
```

Comment:
Think I should combine them?  Is it always the case that they map to the same value?